physionet-django/events/forms.py

```diff
@@ -60,3 +60,17 @@ class EventApplicationResponseForm(forms.ModelForm):
     class Meta:
         model = EventApplication
         fields = ('status', 'comment_to_applicant')
+
+
+class EventAgreementForm(forms.ModelForm):
+    class Meta:
+        model = EventAgreement
+        fields = (
+            'name',
+            'version',
+            'slug',
+            'is_active',
+            'html_content',
+            'access_template',
+        )
+        labels = {'html_content': 'Content'}
```

Comment:
Add help text, e.g.

```suggestion
        labels = {'html_content': 'Content'}
        help_texts = {
            'name': '* The displayed name of the agreement.',
            'version': '* The version number of the agreement.',
            'slug': '* A simple string for use in the URL displaying the agreement. Should include the version number.',
            'is_active': '* Only active agreements are usable in future events.',
            'html_content': '* The agreement text displayed to the participant.',
            'access_template': '* Instructions on accessing the dataset.'
        }
```

Unsure if I guessed correctly at access_template.

Reply:
Thank you. 

I am also not sure about `access_template`. on the fixtures there is no example for `access_template`.

I check with January on what HDN has for `access_template` and turns out we just have the sentence `I accept`.

But your explanation/help text is the best so far.

In any case, i will go with your suggestion for now and ping Tom about the `access_template`